cms/models.py

```diff
@@ -244,10 +244,15 @@ class BootcampRunPage(BootcampPage):
     bootcamp_run = models.OneToOneField(
         "klasses.BootcampRun",
         null=True,
+        blank=True,
```

Comment:
@HamzaIbnFarooq do not forget to create the migration file for this change.

Reply:
Thanks for highlighting that, I just added it.